guides/common/modules/proc_configuring-hosts-to-use-an-alternate-cname-for-content-management.adoc

```diff
@@ -0,0 +1,32 @@
+[id='configuring-hosts-to-use-an-alternate-cname-for-content-management_{context}']
+= Configuring Hosts to Use an Alternate {Project} CNAME for Content Management
+
+If {Project} is configured with an alternate CNAME, you can configure hosts to use the alternate {Project} CNAME for content management. To do this, you must point hosts tot eh alternate CNAME prior to registering the hosts to {Project}. You can do this manually or using the bootstrap script.
```

Comment:
> You can do this manually or using the bootstrap script.

Since below we cover bootstrap first, would it be better to switch order in that sentence?
I mean:

> You can do this with bootstrap script or manually.

Reply:
Good catch, I changed the order of methods because the bootstrap method is automated and seems like the one that users should go for ideally, however they can do things manually too if needed. Will change the order in this sentence too.